internal/app/machined/pkg/system/services/machined.go

```diff
@@ -22,6 +21,65 @@ import (
+	"/machine.MachineService/Events":                       role.MakeSet(role.Admin, role.Reader),
+	"/machine.MachineService/GenerateClientConfiguration":  role.MakeSet(role.Admin),
+	"/machine.MachineService/GenerateConfiguration":        role.MakeSet(role.Admin),
+	"/machine.MachineService/Hostname":                     role.MakeSet(role.Admin, role.Reader),
+	"/machine.MachineService/Kubeconfig":                   role.MakeSet(role.Admin),
+	"/machine.MachineService/List":                         role.MakeSet(role.Admin, role.Reader),
+	"/machine.MachineService/LoadAvg":                      role.MakeSet(role.Admin, role.Reader),
+	"/machine.MachineService/Logs":                         role.MakeSet(role.Admin, role.Reader),
+	"/machine.MachineService/Memory":                       role.MakeSet(role.Admin, role.Reader),
+	"/machine.MachineService/Mounts":                       role.MakeSet(role.Admin, role.Reader),
+	"/machine.MachineService/NetworkDeviceStats":           role.MakeSet(role.Admin, role.Reader),
+	"/machine.MachineService/Processes":                    role.MakeSet(role.Admin, role.Reader),
+	"/machine.MachineService/Read":                         role.MakeSet(role.Admin),
+	"/machine.MachineService/Reboot":                       role.MakeSet(role.Admin),
+	"/machine.MachineService/RemoveBootkubeInitializedKey": role.MakeSet(role.Admin),
+	"/machine.MachineService/Reset":                        role.MakeSet(role.Admin),
+	"/machine.MachineService/Restart":                      role.MakeSet(role.Admin),
+	"/machine.MachineService/Rollback":                     role.MakeSet(role.Admin),
+	"/machine.MachineService/ServiceList":                  role.MakeSet(role.Admin, role.Reader),
+	"/machine.MachineService/ServiceRestart":               role.MakeSet(role.Admin),
+	"/machine.MachineService/ServiceStart":                 role.MakeSet(role.Admin),
+	"/machine.MachineService/ServiceStop":                  role.MakeSet(role.Admin),
+	"/machine.MachineService/Shutdown":                     role.MakeSet(role.Admin),
+	"/machine.MachineService/Stats":                        role.MakeSet(role.Admin, role.Reader),
+	"/machine.MachineService/SystemStat":                   role.MakeSet(role.Admin, role.Reader),
+	"/machine.MachineService/Upgrade":                      role.MakeSet(role.Admin),
+	"/machine.MachineService/Version":                      role.MakeSet(role.Admin, role.Reader),
+
+	"/network.NetworkService/Interfaces": role.MakeSet(role.Admin, role.Reader),
+	"/network.NetworkService/Routes":     role.MakeSet(role.Admin, role.Reader),
+
+	// per-type authorization is handled by the service itself
+	"/resource.ResourceService": role.MakeSet(role.Admin, role.Reader),
+
+	"/storage.StorageService/Disks": role.MakeSet(role.Admin, role.Reader),
+
+	"/time.TimeService/Time":      role.MakeSet(role.Admin, role.Reader),
+	"/time.TimeService/TimeCheck": role.MakeSet(role.Admin, role.Reader),
+}
```

Comment:
I now wonder if we need prefix handling at all. Maybe it would be better to require all methods there in full